cookbooks/aws-parallelcluster-entrypoints/recipes/external_slurmdbd_config.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+#
+# Copyright:: 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with the
+# License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+node.default['cluster']['slurm']['install_dir'] = '/opt/slurm'
```

Comment:
Why not set these in an Attribute file?

Reply:
I am looking at the PR sequentially by the commit, so some of the comments might be based on the initial commit which you might have changed in the last/later commit.

```# Mock necessary attributes to reuse the recipes from ParallelCluster.```
The above comment about mocking the attributes adds the burden of us maintaining the values in 2 places.